zerver/lib/cache.py

```diff
@@ -46,16 +54,17 @@ def remote_cache_stats_finish():
     remote_cache_total_time += (time.time() - remote_cache_time_start)
 
 def get_or_create_key_prefix():
-    # type: () -> str
+    # type: () -> text_type
     if settings.TEST_SUITE:
         # This sets the prefix mostly for the benefit of the JS tests.
         # The Python tests overwrite KEY_PREFIX on each test.
-        return 'test_suite:' + str(os.getpid()) + ':'
+        return 'test_suite:' + text_type(os.getpid()) + ':'
```

Comment:
Hmm, this is a bit confusing, because the LHS and RHS of this computation are `str` still; maybe clearest would be `u"test_suite:%s:" % (os.getpid(),)` ?